internal/util/grpcclient/client.go

```diff
@@ -316,6 +333,21 @@ func (c *ClientBase[T]) ReCall(ctx context.Context, caller func(client T) (any,
 	return ret, err
 }
 
+func (c *ClientBase[T]) bgHealthCheck(client T) {
+	c.sf.Do("healthcheck", func() (any, error) {
+		// v2.2.0 does not has paramtable, use magic nubmer here
+		ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
```

Comment:
As we have change the etcd lease timeout to 20s, it should be 20s here too?

Reply:
60s is for backward compatiblity, say user deployed with older helm version, session TTL may not update to 20s after upgrade